<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderConfigurator</class>
 <widget class="QWidget" name="SenderConfigurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="videoName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Video ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="idSpin"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Video Sender:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="videoSenderSelection"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthSpin">
        <property name="maximum">
         <number>1920</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightSpin">
        <property name="maximum">
         <number>1080</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>IP Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="systemIpAddr">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Config interval:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="configInterval"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>PT:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ptSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Framerate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="framerateSpin"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>UUID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uuidLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="text">
         <string>Restart</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>394</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QCheckBox" name="broadcastEnabled">
        <property name="text">
         <string>Broadcast Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>IP addr:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipAddr"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ipPort">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
